cmd/changelog-parser/testdata/expected_release_output.txt

```diff
@@ -18,59 +18,75 @@ to their releases:
 - **[cyberark/conjur-oss-helm-chart v1.3.7](https://github.com/cyberark/conjur-oss-helm-chart/releases/tag/v1.3.7)** (2019-01-31) [![Certification Level](https://img.shields.io/badge/Certification%20Level-Certified-Green)](https://github.com/cyberark/conjur-oss-helm-chart)
 - **[cyberark/conjur-api-python3 v0.0.5](https://github.com/cyberark/conjur-api-python3/releases/tag/v0.0.5)** (2019-12-06) [![Certification Level](https://img.shields.io/badge/Certification%20Level-Community-Yellow)](https://github.com/cyberark/conjur-api-python3)
 
-## Upgrade Instructions
+# Upgrade Instructions
 
 Upgrade instructions are available for the following components:
 
 - [cyberark/conjur](https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Deployment/Upgrade/upgrade-intro.htm)
 
-## Changes
-
+# Changes
 The following are changes to the constituent components since the last Conjur
 OSS Suite release:
 
-### [cyberark/conjur v1.3.6](https://github.com/cyberark/conjur/releases/tag/v1.3.6) (2019-02-19)
+- [cyberark/conjur](#cyberarkconjur)
+- [cyberark/conjur-oss-helm-chart](#cyberarkconjur-oss-helm-chart)
+- [cyberark/conjur-api-python3](#cyberarkconjur-api-python3)
+
+
+## cyberark/conjur
+
+### [v1.3.6](https://github.com/cyberark/conjur/releases/tag/v1.3.6) (2019-02-19)
+
+* **Changed**
+    - Reduced IAM authentication logging
+
```

Comment:
Stray newline - probably something in template isn't removing whitespaces correctly here